cool.py

```diff
@@ -372,8 +405,13 @@ def locate_results(results, col, row):
     zoom_level = 20
 
     #: get upper-left coordinates of the primary tile (upper-left tile)
-    tile = mercantile.ul(int(col), int(row), zoom_level)
-
+    try:
+        tile = mercantile.ul(int(col), int(row), zoom_level)
+    except ValueError as error:
+        logging.error("error getting tile coordinates on %s, %s: %s", col, row, error, exc_info=True)
+        
+        return results_df
```

Comment:
**question**: this dataframe will be missing the series that are added during a successful location run. Are you sure that is ok for the rest of the processing? Should `None` be returned and skipped later in the pipeline?

Reply:
I'm not sure yet.  I guess we would be appending a partial table (without the centroid columns) into the BigQuery results table, so I'm not sure how it would handle that.